pkg/sqlqueries/SQL/mto/milestone/delete.sql

```diff
@@ -0,0 +1 @@
+DELETE FROM mto_milestone WHERE id = :id;
```

Comment:
Q: Should this have a returning statement? I think we have it both ways in different parts of the app. I think there is some benefit in returning the record, but also fine to leave like this. I'm not sure if the FE ever uses the returned result

Reply:
Same as below, tracked a comment to revisit this!

53a587fe7e2caeacf386dd75fb0c92cc6cece6a0